internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_server_resource_test.go

```diff
@@ -0,0 +1,210 @@
+package systemcentervirtualmachinemanager_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/systemcentervirtualmachinemanager/2023-10-07/vmmservers"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type SystemCenterVirtualMachineManagerServerResource struct{}
+
+func TestAccSystemCenterVirtualMachineManagerServerSequential(t *testing.T) {
+	// NOTE: this is a combined test rather than separate split out tests because only one System Center Virtual Machine Manager Server can be onboarded at a time on a given Custom Location
+
+	if os.Getenv("ARM_TEST_CUSTOM_LOCATION_ID") == "" || os.Getenv("ARM_TEST_FQDN") == "" || os.Getenv("ARM_TEST_PORT") == "" || os.Getenv("ARM_TEST_USERNAME") == "" || os.Getenv("ARM_TEST_PASSWORD") == "" {
+		t.Skip("Skipping as one of `ARM_TEST_CUSTOM_LOCATION_ID`, `ARM_TEST_FQDN`, `ARM_TEST_PORT`, `ARM_TEST_USERNAME`, `ARM_TEST_PASSWORD` was not specified")
```

Comment:
actually, how come these need to be ENV variables and are not part of the config?

Reply:
@katbyte , because creating System Center Virtual Machine Manager Server requires the resources (like SCVMM host machine/Arc Resource Bridge/Custom Location/etc) with additional manually configurations.

Reply:
Is there any way we could spin these up via terraform so we can actually test this resource in teamcity?

Reply:
Creating this resource requires SCVMM host machine that is hosted in the LAB infra (and not as Azure VM).